lib/gitsh/interactive_runner.rb

```diff
@@ -53,6 +58,16 @@ def greeting_enabled?
       env.fetch('gitsh.noGreeting') { 'false' } != 'true'
     end
 
+    def load_gitshrc
+      script_runner.run(gitshrc_path)
+    rescue NoInputError
+      false
```

Comment:
I was uncomfortable just leaving an empty rescue block. Do you think that—and an implicit `nil`—would be preferable to a potentially confusing unused value?